exercises/hamming/canonical-data.json

```diff
@@ -79,6 +79,50 @@
         "strand2": "AGTG"
       },
       "expected": {"error": "left and right strands must be of equal length"}
+    },
+    {
+      "uuid": "5dce058b-28d4-4ca7-aa64-adfe4e17784c",
+      "description": "disallow left empty strand",
+      "property": "distance",
+      "input": {
+        "strand1": "",
+        "strand2": "G"
+      },
+      "expected": {"error": "left strand must not be empty"}
+    },
+    {
+      "uuid": "db92e77e-7c72-499d-8fe6-9354d2bfd504",
+      "description": "disallow left empty strand",
+      "comments": ["Normalises error messages"],
+      "reimplements": "5dce058b-28d4-4ca7-aa64-adfe4e17784c",
+      "property": "distance",
+      "input": {
+        "strand1": "",
+        "strand2": "G"
+      },
+      "expected": {"error": "left and right strands must be of equal length"}
```

Comment:
This appears to be a test that is identical to the test above, just different words for the same situation.  And the reliance that a strand can not be 0 length.

Reply:
Yes, this is a reimplementation (see ln 97 `reimplement` property) of the test on ln 84 that normalises the error message to demonstrate that the logical result is actually the same as non-empty strands of unequal length (such as on ln 81).

Reply:
I see in #97 but it doesn't seem to be related.

Reply:
I see #1762 though.  I think this is what you meant be "In 97"?

Reply:
Nope, that doesn't make sense.  Sorry very confused at the moment.

Reply:
In ln  Sorry, apparently the font on my browser doesn't allow the visual distinction between what I thought was a capital "I" (eye) and a lower case "l" (ell) and though it was a typographical error, wrong case for "in".

Reply:
We really should use "line" when we mean to say "line" to avoid confusion.

Reply:
Sorry, my bad! Will use "line" going forward